README.md

```diff
@@ -1,2 +1,71 @@
-# Book_store
-A react book store app
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magician
+
+![](./src/images/ss1.png)
+
+> A single page app build using react
+
+
+Additional description about the project and its features.
+
+## Tech used
+
+- React
+- Git
+- npm
+- webpack
+
+# Setup WebApp
+
+#### Cloning the project
+
+```
```

Comment:
- [ ] Please, consider fixing the README file, which came from other project (Math Magician) - try to adopt to current project 👍